### DISCUSSION TITLE
Import path completion popover includes system folders

### DISCUSSION THREAD

**Discussion by dimitrisnl** [zed-industries/zed#13978] [Feature Requests]
Why is this happening? Any ideas?

![CleanShot 2024-07-09 at 12 27 41](https://github.com/zed-industries/zed/assets/4951004/4400fb3f-109a-412e-b1e6-88fd061f7f93)

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
